src/SFA.DAS.AdminService.Settings/WebConfiguration.cs

```diff
@@ -1,30 +1,30 @@
-﻿using Newtonsoft.Json;
+﻿using System.Text.Json.Serialization;
 using SFA.DAS.AdminService.Common.Settings;
 
 namespace SFA.DAS.AdminService.Settings
 {
     public class WebConfiguration : IWebConfiguration
     {
-        [JsonRequired] public AzureApiAuthentication AzureApiAuthentication { get; set; }
+        public AzureApiAuthentication AzureApiAuthentication { get; set; }
```

Comment:
Why is this one no longer required/included?

Reply:
It was removed by mistake -- I've re-added it now. I've also replaced several instances of the JsonInclude attribute with JsonRequired, since I think that's more equivalent to Newtonsoft's JsonRequired attribute